src/main/perf/PointsPKLookupTask.java

```diff
@@ -24,17 +24,8 @@
 import java.util.Random;
 import java.util.Set;
 
-import org.apache.lucene.document.IntPoint;
-import org.apache.lucene.index.LeafReader;
 import org.apache.lucene.index.LeafReaderContext;
-import org.apache.lucene.index.PostingsEnum;
-import org.apache.lucene.index.TermsEnum;
-import org.apache.lucene.search.DocIdSetIterator;
 import org.apache.lucene.search.IndexSearcher;
-import org.apache.lucene.search.Query;
-import org.apache.lucene.search.TopDocs;
-import org.apache.lucene.util.Bits;
-import org.apache.lucene.util.BytesRef;
```

Comment:
Thanks to IDE:)
> Maybe we should fail-on-unused-imports like Lucene (checkstyle?)?

It could be hard, since luceneutil directly build against lucene main branch, which is unpublished, I don't know whether we're able to set it up in github to build automatically everytime...